protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -306,6 +310,18 @@ func (k Keeper) UpdateSubaccounts(
 	// Apply the updates to asset positions.
 	UpdateAssetPositions(settledUpdates)
 
+	// Transfer collateral between collateral pools for any isolated perpetual positions that changed
+	// state due to an update.
+	for i, stateTransition := range isolatedPerpetualStateTransitions {
+		if err := k.transferCollateralForIsolatedPerpetual(
+			ctx,
+			settledUpdates[i].SettledSubaccount,
```

Comment:
We pass in `settledUpdates[i].SettledSubaccount` as `updatedSubaccounts` - is this because of logic above
```
	// Apply the updates to perpetual positions.
	UpdatePerpetualPositions(
		settledUpdates,
		perpIdToFundingIndex,
	)
	```
	
	If so, can we comment to make this clear?

Reply:
Yes, that's correct. Added a comment.